
controller:
    joint_state_controller:
        type: joint_state_controller/JointStateController
        publish_rate: 50


    front_left_velocity_controller:
        type: effort_controllers/JointVelocityController
        joint: front_left_wheel_joint
        pid: {p: 100.0, i: 0.01, d: 10.0,}
    back_left_velocity_controller:
        type: effort_controllers/JointVelocityController
        joint: back_left_wheel_joint
        pid: { p: 100.0, i: 0.01, d: 10.0, }
    front_right_velocity_controller:
        type: effort_controllers/JointVelocityController
        joint: front_right_wheel_joint
        pid: { p: 100.0, i: 0.01, d: 10.0, }
    back_right_velocity_controller:
        type: effort_controllers/JointVelocityController
        joint: back_right_wheel_joint
        pid: { p: 100.0, i: 0.01, d: 10.0, }





